src/engine.rs

```diff
@@ -336,17 +336,13 @@ fn print_summary(
 ) {
     println!(
         r#"
-traversed {} files
-matched {} files to formatters
-left with {} files after cache
-of whom {} files were re-formatted
-all of this in {:.0?}
+{} files changed in {:.0?} (processed {}, matched {}, cache misses {}) 
```

Comment:
```suggestion
{} files changed in {:.0?} (found {}, matched {}, cache misses {}) 
```
I'm not sure what processed entails when I read it. Is "found" better?